scaleway/resource_lb_frontend_beta.go

```diff
@@ -108,18 +195,105 @@ func resourceScalewayLbFrontendBetaRead(d *schema.ResourceData, m interface{}) e
 		return err
 	}
 
-	d.Set("lb_id", newRegionalId(region, res.Lb.ID))
-	d.Set("backend_id", newRegionalId(region, res.Backend.ID))
-	d.Set("name", res.Name)
-	d.Set("inbound_port", int(res.InboundPort))
-	d.Set("timeout_client", flattenDuration(res.TimeoutClient))
+	_ = d.Set("lb_id", newRegionalId(region, res.Lb.ID))
+	_ = d.Set("backend_id", newRegionalId(region, res.Backend.ID))
+	_ = d.Set("name", res.Name)
+	_ = d.Set("inbound_port", int(res.InboundPort))
+	_ = d.Set("timeout_client", flattenDuration(res.TimeoutClient))
 
 	if res.Certificate != nil {
-		d.Set("certificate_id", newRegionalId(region, res.Certificate.ID))
+		_ = d.Set("certificate_id", newRegionalId(region, res.Certificate.ID))
 	} else {
-		d.Set("certificate_id", "")
+		_ = d.Set("certificate_id", "")
+	}
+
+	//read related acls.
+	resAcl, err := lbAPI.ListACLs(&lb.ListACLsRequest{
+		Region:     region,
+		FrontendID: ID,
+	}, scw.WithAllPages())
+	if err != nil {
+		return err
+	}
+	sort.Slice(resAcl.ACLs, func(i, j int) bool {
+		return resAcl.ACLs[i].Index < resAcl.ACLs[j].Index
+	})
+	stateAcls := make([]map[string]interface{}, 0, len(resAcl.ACLs))
+	for _, apiAcl := range resAcl.ACLs {
+		stateAcls = append(stateAcls, flattenLbAcl(apiAcl))
+	}
+	_ = d.Set("acl", stateAcls)
+
+	return nil
+}
+
+func resourceLbAclBetaUpdate(d *schema.ResourceData, lbAPI *lb.API, region scw.Region, frontendID string) error {
```

Comment:
in order to keep function name scoping could you rename to `resourceScalewayLbFrontendBetaUpdateAcl`